KsApi/GraphSchema.swift

```diff
@@ -802,3 +848,9 @@ extension Query.Notifications {
     hasher.combine(self.description)
   }
 }
+
+// MARK: - Schema Constants
+
+public extension Query {
+  static let defaultPaginationCount: Int = 25
```

Comment:
Can we use type inference to remove the `Int` here?

Reply:
I've usually had this habit of being explicit with constants about their type, unless you feel strongly I'll keep it? But I could go either way.

Reply:
No all good.